packages/manager/apps/ips/mocks/organisations/organisations.ts

```diff
@@ -1,6 +1,8 @@
 import { Handler } from '@ovh-ux/manager-core-test-utils';
+import { PathParams } from 'msw';
 
-export const organisationMockList = ['ARIN-1', 'ARIN-2', 'RIPE-1', 'RIPE-2'];
+import organisationsList from './get-organisations.json';
+import organisationsDetails from './get-organisations-details.json';
```

Comment:
I think it's better to have it in typescript format instead of json to be able to typecheck it (especially the organisation details list)